.github/checkgroup.yml

```diff
@@ -217,18 +211,15 @@ subprojects:
       - "lite-cpu (macOS-11, lite, 3.9, 1.11)"
       - "lite-cpu (macOS-11, lite, 3.8, 1.10)"
       - "lite-cpu (macOS-11, lite, 3.10, 1.12)"
-      - "lite-cpu (macOS-11, lite, 3.10, 1.13, pre)"
```

Comment:
Where are the new non-pre 1.13 jobs?

Reply:
Good catch, just fixing... but it is example when you update the list only at one place :(